File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int height;

    //check if entered height is in range of 0-23
    do
    {
    printf("Height: ");
    height = get_int();
    }
    while(height < 0 || height > 23);

    //first loop for each row
    for(int i = 0; i < height; i++) {

    //second for blanks
        for(int j = 1; j < height - i; j++) {
            printf(" ");
        }

    //...and third for bricks
        for(int k = 0; k <= i; k++) {
            printf("#");
        }

    //gap
        printf("  ");

    //other side of pyramid
        for(int k = 0; k <= i; k++) {
            printf("#");
        }

    //end of the row
        printf("\n");
    }

}

File: pset6/mario.py
import cs50

# prompt user to enter height that is in range of 1 - 23
while True:
    print('Height: ', end='')
    height = cs50.get_int()
    if height > 0 and height <= 23:
        break

for i in range(height):                     # for each row
    print(' ' * (height - i - 1), end='')   # print blanks
    print('#' * (i + 1), end='')            # print bricks
    print('  ', end='')                     # print gap
    print('#' * (i + 1))                    # finish the row with bricks and newline